apps/communications/dialer/test/marionette/save_contact_name.js

```diff
@@ -0,0 +1,115 @@
+'use strict';
```

Comment:
Well, it is a bit different :) In this test I create the contact with Name, in the other I just save it without modifying it. It was just for practicing. This is the test that runs ok for the bug Bug 1219653 but that is not written in a good way (everything is in the test, the asserts, the waits...and everything...)